dbt/models/qc/docs.md

```diff
@@ -27,16 +27,6 @@ Pulled from the following Inquire queries:
 - `FP Checklist--Bldg and parcel class mismatch` (#2161) - Will C.
 {% enddocs %}
 
-# vw_incorrect_val_method
-
-{% docs view_vw_incorrect_val_method %}
-Test that no residential parcels are using a cost approach.
-
-Pulled from the following Inquire queries:
-
-- `FP Checklist - Res parcels not set to Cost Approach` (#1578) - Will C.
-{% enddocs %}
```

Comment:
This view has been refactored into `qc.vw_report_town_close_res_parcels_not_set_to_cost_approach`.